lib/dealership.rb

```diff
@@ -0,0 +1,48 @@
+class Dealership
+  attr_accessor :inventory, :inventory_count, :details
+
+  def initialize(dealership, address)
+    @dealership = dealership
+    @address = address
+    @inventory = []
+  end
+
+  def add_car(car)
+    @inventory.push(car)
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def cars_by_make(make) # try to refactor without if statement if time.
+    car_match = []
+    @inventory.find_all do |car|
+      if car.make == make
+        car_match.push(car)
+      end
+    end
+  end
+
+  def total_value
+    total = @inventory.map do |car|
+      car.total_cost
+    end
+
+    total = total.inject(:+)  # Instructor: is it possible to reach into the car
```

Comment:
Honestly, I rarely use `.inject` but I can't think of an easy way to do this rather than what you did. Another enumerable you could have used though would have been `.sum` https://apidock.com/ruby/v2_5_5/Enumerable/sum